opencti-platform/opencti-graphql/config/default.json

```diff
@@ -124,6 +124,33 @@
     "next_days": 30,
     "created_by_based": false
   },
+  "protected_sensitive_config": {
+    "enabled": false,
```

Comment:
Is it intended to default to false? Shouldn't it be enabled by default and disabled when the users know what they are doing?

Reply:
it should be disable by default